commcare-cloud/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -310,19 +318,133 @@ def make_parser(self):
                 "Example Usage: --only=riak,stanchion"
             )
         )
+        self.parser.add_argument(
+            '--worker_name',
+            help=(
+                "Celery worker name as appearing in the status list. "
+                "This can also be a comma-separated list of workers"
+                "For ex: Run 'commcare-cloud staging celery status' to get name of workers"
+            )
+        )
 
     def get_inventory_group_for_service(self, service, service_group):
         return self.INVENTORY_GROUP_FOR_SERVICE.get(service, service_group)
 
+    @staticmethod
+    def _get_celery_processes(env):
+        environment = get_environment(env)
+        app_processes_config = environment.translated_app_processes_config
+        return app_processes_config.celery_processes
+
+    def get_celery_config(self, args):
```

Comment:
It looks like the only thing you use from `args` is `args.environment`, so a really simple refactor would be to have this function accept `env` instead, and I think that would be an improvement in that it's more explicit and confined, so easier to reason with.

I would also recommend, though it's a little more work, going one step further and making this (and other methods like `_get_celery_processes`) accept an environment object (from `get_environment(...)`) rather than the env name.

Reply:
i think its a useful extraction to get celery processes out from `env` since its available directly on args. I am guessing this would be happening at more places than 1.  Can move it to `utils` if that makes more sense?